api/server.go

```diff
@@ -42,9 +48,37 @@ func NewServer(config common.Config) error {
 		return errors.New("'org' cannot be empty in the configuration")
 	}
 	Org = config.Org
+	expireTime, err := time.ParseDuration(config.CacheExpireTime)
+	if err != nil {
+		log.Warn("Unexpected error with configured expireTime")
+		// set default expireTime
+		expireTime, _ = time.ParseDuration("4h")
+	}
+	purgeTime, err := time.ParseDuration(config.CachePurgeTime)
+	if err != nil {
+		log.Warn("Unexpected error with configured purgeTime")
+		//set default purgeTime
+		expireTime, _ = time.ParseDuration("15m")
+	}
```

Comment:
Lets check for `""` here first and consider it valid.  that way if you don't include it in the config, it doesn't throw a warning.  Then if you get an error from `ParseDuration` you should return (which will kill the API).